workflow/scripts/add_populations_metadata.py

```diff
@@ -0,0 +1,26 @@
+import zarr
```

Comment:
I'm a bit confused as to what's going on in this script-- I'd expect we'd pass in a csv where each row is a sample, first column is ID (from VCF) and second column is population code. But that doesn't seem to be the case?

Reply:
Yeah I agree this is a bit confusing, but I'm following the lead of the `tsinfer` docs here: https://tskit.dev/tsinfer/docs/latest/usage.html#adding-more-metadata

This script however makes it VERY simple to define the metadata in the config file, where the metadata block looks like this:

```yaml
metadata:
  populations: 
    pop1: 10
```

Reply:
But what if samples aren't in population order in the VCF?

Reply:
ah that would be an issue!

Reply:
i'll change this to accept a csv